src/newsHeadlines.js

```diff
@@ -0,0 +1,38 @@
+let newsArticles = []
+
+function getNewsTitles() {
+  return fetch("http://news-summary-api.herokuapp.com/guardian?apiRequestUrl=http://content.guardianapis.com/search?order-by=newest&show-fields=body&show-elements=image&page-size=20&show-fields=thumbnail")
+    .then( (response) => {
+      return response.json()
+    })
+}
+
+function extractTitles(dataObject) {
+  let results = dataObject.response.results
+
+  for ( let i = 0; i < results.length; i++ ) {
+    let selected = {
+      webTitle: results[i].webTitle,
+    }
+    newsArticles.push(selected)
+  }
+  return newsArticles
+}
+
+  getNewsTitles()
+    .then( (response) => {
+    let titlesHash = extractTitles(response)
+
+    let titlesHTML = renderHTML(titlesHash).join("<br/>")
+    document.getElementById("headlines").innerHTML = titlesHTML
+  })
+
+function renderHTML(data){
+  let headlines = []
+  for ( let i = 0; i < data.length; i++ ){
+    let title = data[i].webTitle
+    let titleHTML = `<p data-titleID="${i}">${title}/p>`
```

Comment:
You're getting a /p> visible at the end of every line - this should be `let titleHTML - '<p data-titleID="${i}">${title}</p>'`

Reply:
I had to change the backticks to single quotes for it to show up correctly in the comment, those are fine it's just the < missing

Reply:
so for the forEach this would be
`data.forEach((x, index) =>
    { let title = x[i].webTitle
    let titleHTML = '<p data-titleID="${i}">${title}/p>'
    headlines.push(titleHTML)
    })` or the let title could be absorbed into the next line. I just think forEachs look so neat compared to for loops